src/pages/community.js

```diff
@@ -72,19 +96,20 @@ const CommunityMemberCard = ({
         onMouseLeave={(e) => (e.currentTarget.style.filter = "grayscale(100%)")}
       >
         <GatsbyImage
-          alt={`${name}'s photo`}
-          image={getImage(img)}
-          style={{
+          alt={`${name}'s photo`}                    // Displays an optimized image using GatsbyImage
+          image={getImage(img)}                      // - Uses the provided img source converted via getImage()
+          style={{                                   // - Applies 100% width and height, and "cover" for object fit
             width: "100%",
             height: "100%",
-            objectFit: "cover",
+            objectFit: "cover",                     
             // borderRadius: "50%",
           }}
         />
       </Box>
     )}
-    {(name?.trim() || role?.trim()) && (
-      <PlainExternalLink href={url} target="_blank">
+    
+    {(name?.trim() || role?.trim()) && (              // Only render the member card if either name or role is non-empty
+      <PlainExternalLink href={url} target="_blank">  // Wraps the card in an external link that opens the provided URL in a new tab 
```

Comment:
Remove this comment too. If you want to, you can later on write the documentation for <PlainExternalLink> component.